test/checkpoint/workers.c

```diff
@@ -263,11 +263,23 @@ real_worker(void)
             goto err;
         } else if (ret == 0) {
             next_rnd = __wt_random(&rnd);
-            if (next_rnd % 7 != 0) {
+            if (next_rnd % 7 == 0) {
```

Comment:
What's the reasoning behind this?
Looks like we'll do timestamped updates less frequently with this change.

Reply:
No, we will commit less frequently with this change. I'm not sure why before we commit in 6/7 chance. I want each commit to have more operations to trigger the bug we see in WT-6733.

Reply:
Yeah, that makes sense.
I lost track of the braces.